travel.html

```diff
@@ -0,0 +1,413 @@
+    </div>
+    
+  <div class="container clearfix  mb-20">
+  <div class="c8">
+  <iframe
+  src="https://map.concept3d.com/?id=1005#!bm/?ct/66558,63891,52803,52750,20079,18027,43513,55127,55129,55130,58109,60907,66559,66728,66730,67878,67879,70457,70458,70459,70460,70461,70462,35364,18014,18012?m/576558?s/?mc/32.8797722038927,-117.23850243115521?z/17?lvl/0"
+  width="750" height="750"></iframe>
+   </div>
+  </div>
+  </div>
+  
+  <h2><span>Travel</span></h2>
+    <h3>Travel to San Diego</h3>
+        <p>The <a href="https://www.san.org/">San Diego International Airport</a> (IATA-code: SAN) is served by a large number of US and international airlines. Direct flights are offered to most major US and many non-US airports. For non-direct flights, the <a href="https://www.flylax.com/">Los Angeles Interinational Airport</a> (IATA-code: LAX) offers direct flights to many locations all over the world, located 193 km (120 mi) from San Diego.
+	</p>
+      <h3>Transport to the Conference Venue</h3>
+
+    <p>Public transportation in San Diego is provided by the
+    <a href="https://www.sdmts.com/">San Diego Metropolitan Transportation Services (MTS)</a> (its website offers a
+    journey planner with timetables for all bus, trolley, and metro services in San Diego).  
+    To reach the UC San Diego from the airport, there are several options:</p>
+
+    <ul class="listing">
+    <li> You can take the 923, 992, or Flyer bus from the airport to the trolley station, and take the blue line north to UCSD Central Campus Station. The cost is approximately $5.
+    </li>
+    &nbsp
+    <li> a taxi (or Uber/Lyft) from the airport costs about $40-$50. </li>
+    </ul>
+      
+    <p>Popular US ride and ride-sharing apps like Uber and Lyft operate in San Diego and will service (pick-up and drop-off) the San Diego International Airport.  The fees are typically ~$40-50 for single rides but can fluctuate due to the demand at the time. The option to automatically share a ride with other Uber/Lyft users will significantly reduce the price.  In order to access these rides, you must install the app on your phone and have access to the internet. Creating an account through the app and registering your credit card is required for service since the drivers will not take cash or credit.
+    </p>
+
+    <h3>Getting around in the city</h3>
+    <p>Public transportation in San Diego is provided by <a href="https://www.sdmts.com">SDMTS</a>. UCSD is a short ride from La Jolla, and about 25-30 minutes from downtown San Diego. The La Jolla beach and surrouding areas are easily accessible by walking.</p>
+
+    <h2><span>Lodging / Hotels</span></h2>
+  	<p>
+	  UC San Diego is located north of downtown La Jolla. Many hotels are available in this area and can be found
+      with any search engine or travel agency. instead to rent an apartment,
```

Comment:
instead -> Instead of renting ...

Reply:
Excellent catch. You inspired me to rewrite that section.